src/Elm/Aeson.hs

```diff
@@ -133,6 +133,7 @@ elmStreetToJson = genericToJSON (elmStreetJsonOptions @a)
 elmStreetJsonOptions :: forall a . Typeable a => Options
 elmStreetJsonOptions = defaultOptions
     { fieldLabelModifier = T.unpack . stripTypeNamePrefix typeName . T.pack
+    , tagSingleConstructors = True
```

Comment:
:open_mouth:

Reply:
Yeah, it has a lot of flags...

Reply:
Maybe it shouldn't be done when newtypes or constructors with fields? Or this is considered here?

Reply:
I hope that this option takes care of this. Anyway, for `newtype` we are using `newtype` strategy so it's not affected...